tests/admin_views/tests.py

```diff
@@ -6181,3 +6182,46 @@ def test_implicitly_generated_pk(self):
         post_data = {'name': '2'}
         response = self.client.post(reverse('admin:admin_views_implicitlygeneratedpk_change', args=(1,)), post_data)
         self.assertEqual(response.status_code, 302)
+
+
+@override_settings(ROOT_URLCONF='admin_views.urls')
+class AddM2MFieldsToObjChangeHistoryTest(TestCase):
```

Comment:
`admin_views` is the place for the test since this is testing a change in the admin views. Putting it near `test_change_view` sounds reasonable.